lib/app_files/category_i_files/home_page.dart

```diff
@@ -350,34 +286,39 @@ class _MainHomePage extends State<MainHomePage> {
-              color: index.isEven
-                  ? AppColors.CONTAINER_WHITE
-                  : AppColors.CONTAINER_COLOR,
-            ),
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                BoldText(text: 'name of the author'),
-                TextHeader(
-                    text: 'name of the author and the title is be here.',
-                    fontColor: AppColors.NORMAL_TEXT_COLOR),
-                SmallText(
-                  text:
-                      'description will be the description will be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the description',
-                  maxLines: 5,
-                )
-              ],
+          GestureDetector(
+            onTap: () {
+              Get.toNamed(FormIntegrator.getDynamicBook(index));
+            },
+            child: Container(
+              //color: Colors.redAccent,
+              width: Dimentions.width350,
+              height: _height,
+              padding: EdgeInsets.all(Dimentions.radius10),
+              margin: EdgeInsets.only(
+                  left: Dimentions.width10, right: Dimentions.width10),
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(Dimentions.radius30),
+                border: Border.all(color: AppColors.CONTAINER_BLACK),
+                color: index.isEven
+                    ? AppColors.CONTAINER_WHITE
+                    : AppColors.CONTAINER_COLOR,
+              ),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  BoldText(text: 'name of the author'),
+                  TextHeader(
+                      text: 'name of the author and the title is be here.',
+                      fontColor: AppColors.NORMAL_TEXT_COLOR),
+                  SmallText(
+                    text:
+                        'description will be the description will be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the descriptionwill be the description',
```

Comment:
these are also dummy text to check the responsive of the implemented code so far so good.